pwa-devdocs/scripts/markdown-linter/tokenizers/jekyll-link.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * A remark-parse tokenizer to add support for jekyll style link definitions 
+ * These definitions have spaces in the link and are not correctly parsed
+ *
+ * For more information see:
+ * - https://github.com/remarkjs/remark/tree/master/packages/remark-parse#extending-the-parser
+ */
+function jekyllLinkDefinition() {
+    var Parser = this.Parser;
+    var tokenizers = Parser.prototype.inlineTokenizers;
+    var methods = Parser.prototype.inlineMethods;
+
+    // Add a new inline tokenizer
+    tokenizers.jekyllLinkDefinition = tokenizer;
+
+    // Run it just before the `link` tokenizer.
+    methods.splice(methods.indexOf('link'), 0, 'jekyllLinkDefinition');
```

Comment:
If there is no `link` tokenizer, do we care when this gets run?

When that's the case, `indexOf` will return `-1` and `splice` will insert `jekyllLinkDefinition` in the 2nd-to-last place in the array (end minus one). Not sure if that matters.

Reply:
Since it's a mouthful and you've already got `jekyllLinkDefinition` as a string here, consider making it a constant at the top of the file.

Then you could use the constant here and on L14, ex: `tokenizers[TOKENIZER_NAME] = tokenizer;`.

Not a huge deal since it's only used in a couple of places.